app/app.go

```diff
@@ -246,16 +247,10 @@ func New(
 	loadLatest bool,
 	invCheckPeriod uint,
 	encodingConfig encoding.Config,
-	upgradeSchedule map[string]upgrade.Schedule,
+	upgradeHeight int64,
```

Comment:
[question] will this only ever be the upgrade height at which celestia-app should upgrade from v1 -> v2? 

Can we clarify via godoc: 
1. this value is only applicable to v1 -> v2 upgraded height
2. what happens if this value is `0`

Reply:
Yes, this will be removed in v3 and can only work when going from v1/v2. It will be ignored for other version.

Reply:
I will add clarification in the godoc